src/pages/LegacySeries/utils.ts

```diff
@@ -6,11 +8,17 @@ import type { EpisodeMetadata } from '#types/series';
 /**
  * Get an array of options for a season filter
  */
-export const getFiltersFromSeries = (playlist: Playlist | undefined): string[] => {
+export const getFiltersFromSeries = (playlist: Playlist | undefined, t: TFunction): string[] => {
   // Old series doesn't have sorting supported and just aggregates all episodes in one playlist
   // So we need to sort the playlist manually based on the selected filter (season).
   return (playlist?.playlist || [])
-    .reduce((filters: string[], item) => (item.seasonNumber && filters.includes(item.seasonNumber) ? filters : filters.concat(item.seasonNumber || '')), [])
+    .reduce(
+      (filters: string[], item) =>
+        item.seasonNumber && filters.includes(item.seasonNumber)
+          ? filters
+          : filters.concat(item.seasonNumber ? `${t('season_prefix')} ${item.seasonNumber}` : ''),
```

Comment:
`season_prefix` should probably change to `season_number_filter_template` with a value of "Season {{season_number}}" or something so we have the ability to support more than "Season #" especially considering that this format won't work for some languages.

Reply:
Nice catch, thank you!